src/design/code.js

```diff
@@ -151,7 +151,8 @@ export const generateJSX = ({
           return !(
             (typeof value === 'object' && Object.keys(value).length === 0) ||
             value === '' ||
-            value === undefined
+            value === undefined ||
+            value === false
```

Comment:
I think this won't work correctly where the value is overriding a default value. For example, `<Clock />` defaults to `run="forward"`. This is turned off by setting `run={false}`.

Reply:
Ah ok that makes sense, shall I remove this part and leave the shorthand part below? Or just reject this whole PR?

Reply:
whichever you’d prefer